### PR TITLE
Translation de-DE

### PR DESCRIPTION
**Pull request by JayMo89**
Updated translations for de-DE

### CODE REVIEW

File: src/locales/de/translation.json [hunk 1]
```diff
@@ -1,185 +1,185 @@
...
     },
     "locationTokens": {
-        "title": "Location Tokens",
-        "mountain": "Mountain",
-        "forest": "Forest",
-        "water": "Water",
+        "title": "Ortsplättchen",
+        "Gebirge": "Mountain",
+        "Wald": "Forest",
+        "Wasser": "Water",
```
Comment (swirle13) [inline src/locales/de/translation.json:67]: All the values on the left hand side should remain unchanged and for translations, the values on the right should be updated. The values on the left are referenced in code in the source language (EN) and are translated into the value on the right, when the language is changed.

Here's my suggested code change you can use to update this:


```suggestion
        "mountain": "Gebirge",
        "forest": "Wald",
        "water": "Wasser",
```

File: src/locales/de/translation.json [hunk 2]
```diff
@@ -1,185 +1,185 @@
...
+        "Vizima": "Wyzima",
         "Vengerberg": "Vengerberg",
         "Cintra": "Cintra",
         "Haern Caduch": "Haern Caduch",
         "Beauclair": "Beauclair",
         "Glenmore": "Glenmore",
         "Doldeth": "Doldeth",
         "Loc Ichaer": "Loc Ichaer",
-        "Gorthur Guaed": "Gorthur Guaed",
+        "Gorthur Guaed": "Gorthur Gvaed",
```
Comment (swirle13) [inline src/locales/de/translation.json:83]: Small fix from Kaligula:

```suggestion
        "Gorthur Gvaed": "Gorthur Gvaed",
```

File: src/locales/de/translation.json [hunk 3]
```diff
@@ -322,106 +322,106 @@
...
                 },
                 {
-                    "title": "Example Type Cards: X (Y/Z) OR X/Z",
-                    "details": [
-                        "Example Type is the type of card, as referred to in Go On Board's \"Components\" lists.",
-                        "X is how many of the cards are found in this pack.",
-                        "Y is the current running total of this type of card across all packs. If excluded, all cards of this type were included in this pack",
-                        "Z is the total number of these cards you should have."
+                    "title": "Beispielkartentypen: X (Y/Z) ODER X/Z",
+                    "details": [
```
Comment (swirle13) [inline src/locales/de/translation.json:409]: Nit: Clean up hanging whitespace, thank you!

```suggestion
                    "details": [
```

File: src/locales/de/translation.json [hunk 4]
```diff
@@ -589,7 +589,7 @@
                 "link": "https://boardgamegeek.com/boardgame/331106/witcher-old-world"
             }
         ],
-        "supportTitle": "Support Me",
+        "supportTitle": "Unterstütze Me",
         "supportLinks": [
             {
                 "text": "Buy me a coffee ☕️",
```
Comment (swirle13) [inline src/locales/de/translation.json:595]: Translation here, as well, please!

**Review by swirle13 — CHANGES_REQUESTED** [review]
Very very good, only a few small changes necessary!

File: src/locales/de/translation.json (hunk 4 above)
Comment (JayMo89) [reply src/locales/de/translation.json:595]:
```suggestion
                "text": "Kauf mir einen Kaffee ☕️",
```

**Review by swirle13 — DISMISSED** [review]
Looks great, didn't find any issues with testing locally. Approved!

**Review by swirle13 — APPROVED** [review]
Looks good! approved

**Pull request merged**